alignak/action.py

```diff
@@ -339,15 +339,15 @@ def copy_shell__(self, new_i):
         return new_i
 
     def got_shell_characters(self):
-        """Check if the command_attribute has shell characters
+        """Check if the command_attribute (command line) has shell characters
         Shell characters are : '!', '$', '^', '&', '*', '(', ')', '~', '[', ']',
                                '|', '{', '}', ';', '<', '>', '?', '`'
 
         :return: True if one shell character is found, False otherwise
         :rtype: bool
         """
-        for command in self.command:
-            if command in SHELLCHARS:
+        for character in self.command:
```

Comment:
in `SHELLCHARS` we have 18 characters and in command more, so it will be more performant to do loop on `SHELLCHARS` and check if the special char is on the command